### DISCUSSION TITLE
Are we missing any key modelers or communites?

### DISCUSSION THREAD

**Discussion by mankoff** [IACS-cryo/Delineation-WG#32] [Modeling]
I just sent an email to everyone who selected "Models" on the [Team Member spreadsheet](https://docs.google.com/spreadsheets/d/18DBHRbEAuquOqChHN2GqUR2uPKpW7D1Y-IBqMPM6TtM/edit#gid=1372946030) asking them to fill out details about their model masks (see template at https://github.com/IACS-cryo/Delineation-WG/issues/new/choose

I think our community and the email request includes:
+ [X] PISM
+ [X] MAR
+ [X] RACMO
+ [X] HIRHAM
+ [X] BedMachine
+ [X] ISSM
+ [X] CISM
+ [x] MetUM
+ [ ] COSMO-CLM
+ [x] NHM-SMAP - email sent, not yet joined
+ [x] Elmer/Ice - email sent, not yet joined
+ [x] BISICLES - email sent, not yet joined

Are there other individuals, groups, tools, or communites we should be including?

**Comment by NicolajHansen90** [comment]
What about the COSMO-CLM group, or the NHM–SMAP group, or MetUM

**Reply by mankoff** [reply]
@NicolajHansen90 - I don't see much COSMO-CLM cryosphere on Google other than your paper. Do you have a contact for them? I know Masashi and will contact him re NHM-SMAP. Do you have contacts at MetUM?

**Reply by NicolajHansen90** [reply]
For MetUM try Andrew Orr  (anmcr@bas.ac.uk) or Ella Gilbert  (ellgil82@bas.ac.uk).
You might be right about COSMO-CLM, but maybe try contact Niels Souverijns although I am not sure if he still does Polar science

**Comment by bearecinos** [comment]
Perhaps is also worth identifying the models in here: https://www.nature.com/articles/s41586-021-03302-y that had some Antarctic Islands included in their outputs and list them as issues too?

from the paper:
> The Antarctic periphery glaciers (region 19) are
located only on the surrounding islands, not on the mainland ice sheet;
ice sheet models include some of the larger islands, so there is some
overlap in area, but the effect of this is estimated to be small (see Methods
section ‘Comparison with IPCC assessments’ for an estimate of this
and other limitations).

Models here include BISICLES for example

**Reply by mankoff** [reply]
Mention of BISICLES made me also think of Elmer/Ice. I've contacted them. I cannot find a contact for BISICLES. If anyone knows them, please email me.

**Reply by JonathanCarrivick** [reply]
Stephen Cornford <S.L.Cornford@bristol.ac.uk> is a BISICLES developer and user involved with several ongoing projects....
